MetriportSDK.podspec

```diff
@@ -0,0 +1,15 @@
+Pod::Spec.new do |s|
+  s.name             = 'MetriportSDK'
+  s.version          = '0.1.0'
+  s.summary          = 'A Swift Library for access to Apple Healthkit.'
+
+  s.homepage         = 'https://github.com/metriport/metriport-ios-sdk'
+  s.license          = { :type => 'MIT', :file => 'LICENSE.md' }
```

Comment:
What license do we use for this?

Reply:
Should i use same one in OSS?

Reply:
Let's use MIT for this so there aren't limitations in cx implementations.